packages/block-editor/src/components/panel-color-settings/index.native.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { PanelBody, UnsupportedFooterControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const PanelColorSettings = () => {
```

Comment:
PanelColorSettings just uses PanelColorGradientSettings, it may be a good idea to add this code to PanelColorGradientSettings so we cover all use cases of colors and gradients.

Reply:
👍